.travis.yml

```diff
@@ -13,6 +13,7 @@ node_js:
   - "iojs-v1"
   - "iojs-v2"
   - "iojs-v3"
+  - "4.0"
```

Comment:
It should be `4`, not `4.0`. Node will surely get a lot of minor updates before it goes LTS and node-sass should test on the latest of them.

This is analogous to how in the above lines you can see `iojs-v3`, not `iojs-v3.0`.